app/views/articles/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+
+
+<div style="margin-top: 150px;" class="d-flex justify-content-md-center align-items-center ">
+  <div style="border:1px solid black; padding:100px; border-radius: 10px;">
+    
+  <%= form_with model: @article, local: true do |form| %>
```

Comment:
do not use an instance variables in partials, always use locals. Doing that will make your code generic and the same partial can be used in other scenarios as wll.